src/DIRAC/Resources/Computing/AREXComputingElement.py

```diff
@@ -0,0 +1,625 @@
+########################################################################
+# File :   Originally ARCComputingElement.py
+# Author : Raja Nandakumar
+# Use the ARC REST interface and retire api libraries when complete
+########################################################################
+
+""" AREX Computing Element (ARC REST interface)
+    Using the REST interface now and fail if REST interface is not available.
+    A lot of the features are common with the API interface. In particular, the XRSL
+    language is used in both cases. So, we retain the xrslExtraString and xrslMPExtraString strings.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__RCSID__ = "$Id$"
+
+import six
+import os
+import stat
+import sys
+
+import requests
+import json
+import ldap3
+
+# from urllib.parse import urljoin
+
```

Comment:
```suggestion
```

Reply:
Will do as I mentioned to @fstagni a short while ago (at another point)

Reply:
Turns out, this will join at most one part to a URL.

With my prehistoric programming ability, I decided to write an internal function to do this.